src/core/data-tests.js

```diff
@@ -23,11 +23,42 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  
+  var [testFile] = new URL(href).pathname.split("/").reverse();
+  
+  const testParts = testFile.split(".");
+  [testFile] = testParts;
+  
+  const isSecureTest = testParts.find(part => part === "https");
+  const isManualTest = testFile.split(".").join("-").split("-").find(part => part === "manual");
+  
+  let emojiList = document.createElement("span");
+
+  if (isSecureTest) {
+    const requiresConnectionEmoji = hyperHTML.bind(document.createElement("span"))` 🔒 `;
+    requiresConnectionEmoji.setAttribute("aria-label","requires a secure connection");
+    
+    console.log(testFile)
+    testFile.split(".https").join("");
```

Comment:
this line doesn't do anything :)

Reply:
It does. It removes the .https parts from the end